setup.py

```diff
@@ -3,14 +3,20 @@
     from setuptools.command.build_py import build_py as build
     from setuptools.command.build_ext import build_ext
     from setuptools.command.install import install
+    from setuptools.command.clean import clean
```

Comment:
That's a good question. `setuptools` does some amount of monkeypatching, so perhaps `from setuptools import setup, Extension` is enough.